VERSION

```diff
@@ -1 +1 @@
-0.1.0
+0.1.1
```

Comment:
Where is it used? I noticed it is still 0.1.0 in the upstream repo.

Reply:
It's not used afaik; it's more of a point of documentation. Upstream is still at 0.1.0, but I think we are proceeding to 0.1.1 since we release Docker images that need to be versioned correctly